### DISCUSSION TITLE
How to incorporate BTF into my application

### DISCUSSION THREAD

**Discussion by havefuncoding1** [aquasecurity/btfhub#105] [Q&A]
Hi, at end of this [doc](https://github.com/aquasecurity/btfhub/blob/main/docs/generating-tailored-btfs.md), it mentioned BTF files can be incorporated into applications. Are there instructions available for this? Thanks.

**Comment by rafaeldtinoco** [comment]
@havefuncoding1 take a look at https://github.com/rafaeldtinoco/example and let me know if that fits your purposes. There is an example using C with libbpf and another one with Golang with libbpf and libbpfgo. I hope it helps.

**Reply by havefuncoding1** [reply]
Thank you so much, @rafaeldtinoco. This really helps.
